### PR TITLE
Stability and fixes Close #263, #278, #276, #275, #290

### LINKED ISSUE #263: New pokestop icons hard to see.
Is it just me or are the new pokéstop icons very hard to see. The blue is too similar with the search area circule if it's on.  The old red ball was better .

And do you have plans of including satellite view in the map?

### LINKED ISSUE #275: Countdown timer on marker tooltip does not display correctly when single digit seconds are left.
If a pokemon marker has any time left where the number of seconds is zero through nine it will not display correctly. Where it would show 10:05 it instead shows 10:5

examples: 
Expires in: 8:8
Expires in 0:5

screenshot_20160727-192018 1

### LINKED ISSUE #276: Fix Pokestops.
Pokestops won't show farther than 3steps away while they could normally go farther, is this a bug or intentional? I'm confused as to why it does that. (Even if I set the steps at 5, it would still show only 3steps away)

### LINKED ISSUE #278: Graphical issues while scanning.
If I long press to scan in an area while another area is being scanned, it will cause various graphical issues.

### LINKED ISSUE #290: Pokemon number showing wrong
I scanned a area and it should only 1 pokemon but the below bar where scanning text is shown it was showing 5 pokemon because I think I have filtered rattata and pidegy.. but it should show right number of pokemon even after filter.

### PR DESCRIPTION
**Pull request by socrates2510**
# Improve scanning locations issues:
- Change color to yellow instead of pokestop color cyan Close #263
- Remove the final big circle Close #278
- Fix distance for Gyms and Pokestops  Close #276
- Improve count of pokemon caught during a batch of searches - Close #290
- Fix expiration of pokemon in all languages to format 0:00 Close #275
- Markers made then to show while still pending searches
- Merge SearchParams into MapHelper

### CODE REVIEW

**Comment by fess89** [comment]
why do you go with static variables? isn't it better to use NianticManager.getInstance() instead?

**Comment by socrates2510** [comment]
True that. It was a refactoring of where it was (view) to where it belongs (controller) but I hated the them were static. I can change it.

**Comment by fess89** [comment]
could possibly use something like NianticManager.getInstance().incrementPendingSearch(), not make the variables public...

File: app/src/main/java/com/omkarmoghe/pokemap/controllers/net/NianticManager.java [hunk 1]
```diff
@@ -332,8 +334,10 @@ public void run() {
                     Log.e(TAG, "Failed to fetch map information via getCatchablePokemon(). PoGoAPI crashed. Raised: " + e.getMessage());
                     EventBus.getDefault().post(new InternalExceptionEvent(e));
                 }
+                NianticManager.getInstance().currentScan++;
```
Comment (comann) [inline app/src/main/java/com/omkarmoghe/pokemap/controllers/net/NianticManager.java]: Why not use 

```
currentScan++;
```

**Comment by omkarmoghe** [comment]
fixes #263 #278 #276 #190 #275

**Pull request merged**